### PR TITLE
feat(conversation): edit conversations tags

### PR DESCRIPTION
**Pull request by joshikeno**
re [#2812](https://github.com/opencollective/opencollective/issues/2812)

- [x] Implement UI feature
- [x] E2E Test

tags-testing

### CODE REVIEW

File: test/cypress/integration/17-conversations.test.js [hunk 1]
```diff
@@ -39,6 +39,13 @@ describe('Conversations', () => {
         '<div>Hello from <a href="https://opencollective.com/opencollective">https://opencollective.com/opencollective</a> 👋👋👋</div><div>Lorem ipsum dolor sit amet, consectetur adipiscing elit. De hominibus dici non necesse est. Immo alio genere; Si longus, levis; Quicquid enim a sapientia proficiscitur, id continuo debet expletum esse omnibus suis partibus.</div>',
       );
 
+      // Edit tags
+      const sampleTag = 'alot more amazing stuff';
+      cy.getByDataCy('InlineEditField-Trigger-tags').click();
+      cy.getByDataCy('styled-input-tags').type(`${sampleTag}{enter}`);
+      cy.getByDataCy('InlineEditField-Btn-Save').click();
+      cy.contains(`${sampleTag}`.toUpperCase());
+
```
Comment (Betree) [inline test/cypress/integration/17-conversations.test.js]: Nice!

**Comment by joshikeno** [comment]
testing-tags-editor

**Comment by joshikeno** [comment]
@Betree Is the ability to clear tags require?

ezgif com-video-to-gif (1)

**Comment by Betree** [comment]
> testing-tags-editor

Tags should be displayed in the input, not above. It's like when you edit a text field, you expect the input to be pre-filled with the existing value(s). Using `suggestedTags` is not the way to go, as the name indicates this prop is meant to **suggest** tags. What you want is to prefill the field with `defaultTags`. react-select has a `defaultValue` prop, see https://react-select.com/props.

Peek 17-01-2020 13-29

> Is the ability to clear tags required?

I think it's nice, but it's optional. Choose the one you prefer :slightly_smiling_face:

**Comment by joshikeno** [comment]
conversation_tags_edit

**Comment by joshikeno** [comment]
Screenshot 2020-01-20 at 14 10 05

https://opencollective-frontend-euithejaqo.now.sh/engineering/conversations/hello-open-collective-engineering-r4pzbpbm

**Comment by Betree** [comment]
Perfect, I'm adding a small commit for styles and we'll be good to go :taco:

**Comment by Betree** [comment]
I reduced the min-width for action buttons so that they can fit in one line:

image

**Pull request merged**